BungeeCord-Patches/0011-Optimize-uuid-conversions.patch

```diff
@@ -240,6 +244,51 @@ index 86a0055..6c9c6d6 100644
 +        return UUIDUtils.fromString(uuid);
      }
  }
+diff --git a/api/src/main/java/net/md_5/bungee/api/ServerPing.java b/api/src/main/java/net/md_5/bungee/api/ServerPing.java
+index 27b51849..c2209174 100644
+--- a/api/src/main/java/net/md_5/bungee/api/ServerPing.java
++++ b/api/src/main/java/net/md_5/bungee/api/ServerPing.java
+@@ -3,6 +3,8 @@ package net.md_5.bungee.api;
+ import java.util.ArrayList;
+ import java.util.List;
+ import java.util.UUID;
++
++import io.github.waterfallmc.waterfall.utils.UUIDUtils;
+ import lombok.AllArgsConstructor;
+ import lombok.Data;
+ import lombok.Getter;
+@@ -75,7 +77,8 @@ public class ServerPing
+ 
+         public String getId()
+         {
+-            return uniqueId.toString().replaceAll( "-", "" );
++
++            return UUIDUtils.undash( uniqueId.toString() );
```

Comment:
Could you use the fully-qualified class name here, to remove the need for the import?